### ISSUE TITLE
[Feature Request][Battlegrounds] Show achievement status when choosing hero

### ISSUE THREAD

**Issue by JoelMahon** [HearthSim/Hearthstone-Deck-Tracker#4369]
After being shown 4 heroes odds are I will not remember which ones are missing 1st place achievements! It'd be great if an indication of the achievement status was shown.

A circle/icon above/below the hero with either a fill of nothing (never played), Bronze (played but never won), Silver (won but never 1st), or Gold (got 1st) would be my tacky looking solution but I'm sure the creators of this sleek app could whip up something nicer.

Some heroes also have additional achievements, possible worth listing, though they're much less effort to track manually.

**Comment by Bewarebear** [comment]
Yes please! I just came to suggest this, but Joel got to it before me. This would be an excellent addition. :)

**Comment by VarnaScelestus** [comment]
Please this. +1. I was hoping to find something like this but searching online finds nothing. I already use this excellent tracker and it would be icing on the cake to have.
